### ISSUE TITLE
Can't open files via vscode-remote URL handler

### ISSUE THREAD

**Issue by lalten** [microsoft/vscode-remote-release#4902]
I can't open a file via the `vscode://` URL handler if the file is in a remote workspace. 

Works:
```
vscode-insiders://file/Users/user/repo/file.ext
```

Works, but only opens the workspace, not a specific file:
```
vscode-insiders://vscode-remote/ssh-remote+myubuntubox/home/user/repo/.editor/repo.code-workspace
```

Expected to work, but does not work:
```
vscode-insiders://vscode-remote/ssh-remote+myubuntubox/home/user/repo/file.ext
```

Calling from CLI works fine:
```
code-insiders --file-uri=vscode-remote://ssh-remote+myubuntubox/home/user/repo/file.ext
```

My end goal is to register iTerm2 triggers to create hyperlinks for file locations that when clicked open the file in my remote workspace.

- VSCode Version: `1.56.0-insider`
- Local OS Version: `macOS Catalina 10.15.7`
- Remote OS Version: `Ubuntu 20.04.2`
- Remote Extension/Connection Type: `SSH`

Relevant issues I could find:
 * Finding ssh-remote name: https://github.com/microsoft/vscode/issues/78491
 * `--file-uri=vscode-remote://` CLI option: https://github.com/microsoft/vscode/issues/79208
 * https://github.com/microsoft/vscode/issues/108257
 * https://github.com/microsoft/vscode-remote-release/issues/4333

**Comment by aeschli** [comment]
It's a dup of #108257, right?

**Comment by lalten** [comment]
I guess you are right! Maybe you could consider reopening it?

**Comment by aeschli** [comment]
It didn't get enough interest at the time.
If you are interested in creating a PR. My suggestion would be we use the line/offset notation to signal that a path is a file

`vscode-insiders://vscode-remote/ssh-remote+myubuntubox/home/user/repo/file.ext#1,1`

**Comment by aeschli** [comment]
duplicate of microsoft/vscode#108257,

**Comment by aeschli** [comment]
Fixed through microsoft/vscode#108257
